proto/sge/bet/tx.proto

```diff
@@ -37,6 +37,9 @@ message MsgSettleBet {
 
   // bet_uid is the unique uuid of the bet to settle
   string bet_uid = 2 [(gogoproto.customname) = "BetUID" ,(gogoproto.jsontag) = "bet_uid", json_name = "bet_uid"];
+
+  // bettor_address si sthe bec32 address of the bettor
+  string bettor_address = 3;
```

Comment:
This message can be called by any user (with any address different from the betting owner). So the `creator` is the address of the account that will settle a bet, and the bettor's address is the address of the betting owner. 
We need the bettor address because of the latest changes in the KVStore design and the inclusion of `creator` in the key generate.